.github/workflows/ci.yaml

```diff
@@ -67,3 +68,49 @@ jobs:
 
       - name: Check format
         run: ./scripts/check_fmt.sh
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          # Needed to get older tags
+          fetch-depth: 0
```

Comment:
we probably don't need the tags if we're just building a preview image on `main`?